src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -88,7 +89,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $io = new SymfonyStyle($input, $output);
 
         if (null === $twig = $this->getTwigEnvironment()) {
-            $io->error('The Twig environment needs to be set.');
+            $errorIo = $output instanceof ConsoleOutputInterface ? new SymfonyStyle($input, $output->getErrorOutput()) : $io;
+            $errorIo->error('The Twig environment needs to be set.');
```

Comment:
BTW, I've reverted a similar change in my previous PR thinking it was the same, but actually it's another place: https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Twig/Command/DebugCommand.php#L90